curvefs/docker/debian10/Dockerfile

```diff
@@ -1,11 +1,15 @@
 FROM opencurvedocker/curve-base:debian10
 ENV TZ=Asia/Shanghai
 RUN mkdir -p /curvefs /etc/curvefs /core /etc/curve
-COPY curvefs /curvefs
+COPY curvefs/client /curvefs
```

Comment:
I think only one OS Dockerfile needs to be kept for running curve in docker (maybe debian9 or debian10), it's too many to maintain.

Reply:
> I think only one OS Dockerfile needs to be kept for running curve in docker (maybe debian9 or debian10), it's too many to maintain.

removed